libs/qCC_db/include/ccCoordinateSystem.h

```diff
@@ -0,0 +1,111 @@
+//##########################################################################
+//#                                                                        #
+//#                              CLOUDCOMPARE                              #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 or later of the License.      #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#          COPYRIGHT: Chris Brown                                        #
+//#                                                                        #
+//##########################################################################
+
+#pragma once
+
+//Local
+#include "ccGenericPrimitive.h"
+
+class ccPlane;
+
+//! Coordinate System (primitive)
+/** Coordinate System primitive
```

Comment:
Maybe not necessary ;)

Reply:
actually it is because I later return planes from this primitive so it needs the forward declaration

   ``` c++
    ccPlane getXYplane() const;
	ccPlane getYZplane() const;
	ccPlane getZXplane() const;
```